blocks/embed/embed.css

```diff
@@ -56,6 +56,168 @@
     left: 7px;
   }
 
+  /* VIMEO STYLES */
+lite-vimeo {
+  aspect-ratio: 16 / 9;
+  background-color: var(--laz-black);
+  position: relative;
```

Comment:
Is it possible to fix the missing thumbnail ? check: https://www.learninga-z.com/site/resources/videos/teacher-testimonials-lohse ?